common/src/main/java/io/homeassistant/companion/android/common/sensors/SensorReceiverBase.kt

```diff
@@ -299,6 +299,7 @@ abstract class SensorReceiverBase : BroadcastReceiver() {
                         sensor.registered = null
                         sensor.lastSentState = ""
                         sensor.lastSentIcon = ""
+                        sensor.forceUpdate = false // TODO is this correct?
```

Comment:
I need assistance with this. What do you think, should we set it to `false` here?